src/internal/utils/version.go

```diff
@@ -101,11 +166,13 @@ func UpdateLastModified(metadata map[string]interface{}) {
 
 // VersionWarning prints a warning if the version is has known issues.
 func VersionWarning(version string) {
```

Comment:
Is the expectation for downstream consumers to replicate behavior similar to this function?

Reply:
It executes when creating a new validator. So the consumer doesn't need to do anything and it will apply when running revision or validation.

Reply:
This was existing behavior. I just added the new warning for latest version. Previously it was just the warning for 1.0.5.

Reply:
For consuming projects who have opinionated logging (IE Lula and it's various log levels and logging to file) I wonder if there is a way to defer that optionality to the consumer. That explicit log may not align with logging requirements/format/expectations). agree/disagree?

Reply:
Yeah I Agree with that. I will extract that and have it return a a message? or an error? what do you think would be a good route for that. Or just create a helper method. This would apply to the 1.0.5 error as well, should we bother with the messaging in go-oscal or shift that with the helper methods like the GetLatestVersion()

Reply:
I believe there is a good balance to be had here. Go-OSCAL the CLI should have the functionality implemented such that the library functionality exposes what it (and consumers) need to write CLI commands. The go-oscal CLI itself can serve as a useful tool as well as an example implementation for other OSCAL tools (logging should be done at the CLI layer). 

A healthy split should provide more pathways to understanding how a change in go-oscal impacts consumers when it comes to CLI functionality. This very well could change where we import functionality from and may require re-wiring across the library, but feels like a strong stance overall.

Reply:
I agree I think I am just looking for that split. For me I am thinking thinking anything in the cmds folder is likely not something I would expect to be used by a consumer. Though all of it is kind of exposed. I would also like to (iteravely) extract a lot of the utility methods into public packages, though they are currently able to be imported as needed.